src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -39,6 +43,24 @@ function isNative(fn) {
   }
 }
 
+type Item = {
+  element: ReactElement,
+  parentID: DebugID,
+  text: ?string,
+  childIDs: Array<DebugID>,
+  isMounted: bool,
+  updateCount: number,
+};
+
+declare function getItem(id: DebugID): ?Item;
```

Comment:
If we want to start using `let`/`const` consistently instead of `var`. We would have to hoist the statements upwards.

Can you do something like this?

```js
let foo : (x : number) => string;

if (window.bar) {
  foo = (x : number) => x + 'bar';
} else {
  foo = (x : number) => x + '';
}

let bar : string = foo(10);
```

Flow is smart enough to know that it is not possible to be `undefined` here.